app/controller/sdl/Abstract/Controller.js

```diff
@@ -1758,6 +1758,53 @@ SDL.SDLController = Em.Object.extend(
       SDL.SDLModel.webApplicationFramesMap[policyAppID].hidden = false;
     },
 
+    /**
+     * @function onUpdateFile
+     * @param {String} fileName
+     * @description Checks whether provided file was requested from SDL or not
+     * and sends notification if file was not requested yet
+     */
+    onUpdateFile: function(fileName) {
+      if(SDL.SDLController.model && SDL.SDLController.model.appID) {
+        let model = SDL.SDLController.model;
+        if (model.cachedIconFileNamesList.indexOf(fileName) < 0) {
```

Comment:
```suggestion
        if (!model.cachedIconFileNamesList.includes(fileName)) {
```
might be clearer since we only want to check if that fileName exists in the list

Reply:
@ShobhitAd fixed in https://github.com/smartdevicelink/sdl_hmi/pull/460/commits/fc0ec0196e42cd5a8b20429f28b7b030fdb7c894